algosdk/template.py

```diff
@@ -0,0 +1,280 @@
+import math
+import random
+from . import error, encoding, constants, transaction
+import base64
+
+
+class Template:
+    def get_address(self):
+        """
+        Returns the address of this contract.
+        """
+        p = constants.logic_prefix + base64.b64decode(self.get_program())
+        addr = encoding.checksum(p)
+        return encoding.encode_address(addr)
+    def get_program(self):
+        pass
+
+
+class Split(Template):
+    """
+    Split allows to lock assets in an account which allows to transfer 
+    to two predefined addresses in a specific M:N ratio. Note that the ratio is 
+    specified by the first address part. For example - If you would like to have 
+    a split where the first address receives 30 percent and the second receives 70, 
+    set ratn and ratd to 30 and 100, respectively. 
+    Split also have an expiry round, in which the owner can transfer back the assets.
+    ...
+
+    Arguments:
+        owner (str): an address that can receive the asset after the expiry round
+        receiver_1 (str): first address to receive assets 
+        receiver_2 (str): second address to receive assets 
+        ratn (int): the numerator of the first address fraction
```

Comment:
The comments are updated in: https://github.com/algorand/go-algorand/pull/536/files
May need to be updated here. 
Also, is this just for assets?